lisa/analysis/tasks.py

```diff
@@ -671,10 +656,10 @@ def plot_tasks_wakeups(self, target_cpus=None, time_delta=0.01, axis=None,
         if target_cpus:
             df = df[df.target_cpu.isin(target_cpus)]
 
-        df = self._df_discretize_series(df["target_cpu"], time_delta, "Wakeup count")
-
-        df.plot(ax=axis, legend=False)
+        series = series_rolling_apply(df["target_cpu"], lambda x: x.count(),
```

Comment:
`lambda x: x.count()` could be updated with `series_mean` so that the values that the magnitude of the output does not depend on `time_delta`, and is actually in unit of `x per seconds`.

Reply:
I think you're right, however `series_mean` gives me funny results (because it's average the contents of the series rather than the count). I think what I want here is

```
lambda x: x.count() / (x.max() - x.min())
```

Reply:
right, `x` must be a series of bool in order to use `series_mean()`, in which case the result the same as your formula.